_data/sites.json

```
@@ -1090,6 +1090,19 @@
         ]
     },
 
+    {
+        "name": "AltTextGeneratorAI",
+        "url": "https://alttextgeneratorai.com/terms-of-use",
+        "difficulty": "hard",
+        "notes": "Send us an email at balt1794@gmail.com with the subject and body of Account Deletion Request. After receiving the email, your account will be deleted in the next 24 hours.",
```

```suggestion
        "notes": "Send then an email. After receiving the email, your account will be deleted in the next 24 hours.",
```
No need to mention the email address, not the subject or body, since the button will auto fill that